apps/pie-docs/src/assets/styles/components/_card.scss

```diff
@@ -0,0 +1,67 @@
+@use '../fozzie-config' as f;
+
+$card-width: 328px;
+$card-image-height: 180px;
+$card-label-height: 72px;
+
+.c-card-wrapper {
+    container-type: inline-size;
```

Comment:
nice one ⚡ 
https://www.youtube.com/watch?v=ZSaAHb5dRwQ

Reply:
I absolutely love that we're finding a legit use case for container queries!

Looking at [caniuse](https://caniuse.com/css-container-queries) and comparing with our [browser support guidelines](https://www.pie.design/engineers/guidelines/browser-support/) we should just triple check we're happy with the level of support, and consider whether or not we need to figure out a fallback. 

@ashleynolan would be great to hear your thoughts on this too :)

Reply:
I checked this yesterday – I'm happy with it being used on the docs site. I don't think we should use them in any production customer components/sites yet, mainly because they only just got simplemented in by the Samsung Internet browser last week, and that's a big browser in some of our markets.

Reply:
🎉